### PR TITLE
Change header on month change

### PR DESCRIPTION
**Pull request by roygrv21**
### Requirements

Changing header based on the change of months such as July, September-November, December.

### Description of the Change

Fetching the current month and rendering the headers based out of it.  The implementation is done in a generic way such that headers can also be added later based on the left out months.

### Benefits

This PR would make the website attractive as it would have seasonal changes.

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -13,11 +13,71 @@ export default () => (
...
-      /static/banners/home.jpg"
-    />
+
+    {/* Switch stamement takes the number of month as an expression.
+  More number of cases can be added based on new banners that are supposed
+  to be added for the months that are left out.
+   */}
+
+    {(() => {
+      let abc = 123;
```
Comment (johnnolan) [inline pages/index.js:23]: Looks good! Think you could remove the abc variable and make monthNumber a const?

Comment (roygrv21) [reply pages/index.js]: I have removed the abc variable and changed monthNumber into a const.

File: pages/index.js [hunk 2]
```diff
@@ -13,11 +13,70 @@ export default () => (
...
+        case 6:
+          return (
+            <Header
+              title={'DDD East Midlands'}
+              banner="
+        /static/banners/pride.png"
+            />
+          );
+          break;
+        case 9:
```
Comment (johnnolan) [inline pages/index.js:34]:
```suggestion
        case 9:
        case 10:
        case 11:
          return (
            <Header
              title={'DDD East Midlands'}
              banner="
        /static/banners/autumn-header.jpg"
            />
          );
          break;
```

Can probably do something like the above too reducing the amount of code needed. Last one, promise!

**Review by johnnolan — CHANGES_REQUESTED** [review]

File: pages/index.js (hunk 2 above)
Comment (roygrv21) [reply pages/index.js:34]: Haha! Done.

Comment (johnnolan) [reply pages/index.js:34]: Glourious!

**Review by johnnolan — APPROVED** [review]

**Pull request merged**